cmd/watt/aggregator.go

```diff
@@ -146,7 +146,6 @@ func (a *aggregator) isComplete(p *supervisor.Process, watchset WatchSet) bool {
 			p.Logf("initialized k8s watch: %s", w.WatchId())
 		} else {
 			complete = false
-			p.Logf("waiting for consul watch: %s", w.WatchId())
```

Comment:
Do we want to keep this log statement? Didn't it give us the clue that let us track the issue down?

Reply:
Ack you're right. Errant delete because I had some debug logging right next to it.